test/test_KubeLibrary.py

```diff
@@ -334,12 +334,12 @@ def test_get_role_bindings_in_namespace(self, mock_lnp):
         role_bindings = kl.get_role_bindings_in_namespace('default')
         self.assertEqual(['read-pods'], [item for item in role_bindings])
 
-    @mock.patch('kubernetes.client.CoreV1Api.list_namespaced_config_map')
```

Comment:
Same here, replace test instead of adding

Reply:
Changes Done.